docs/index.bs

```diff
@@ -2875,6 +2875,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
               1. Wait for |task| to have executed or been discarded.
               1. Wait for the step labeled *WaitForAsynchronousExtensions* to complete.
+              1. If |installingWorker|'s [=list of router rules=] [=list/contains=] "fetch-event" and [=set of event types to handle=] does not [=set/contain=] <code>fetch</code>, then set |installFailed| to true.
```

Comment:
"set of event types to handle" is a field of the service worker type. So we need to reference a worker. I think it is `|installingWorker|`? So you would say `|installingWorker|'s [=set of event types to handle=]`.

Reply:
done